### PR TITLE
Update Mandala endpoint

### PR DESCRIPTION
**Pull request by pgolovkin**
Update Mandala endpoint

### CODE REVIEW

File: chains/v3/chains_dev.json [hunk 1]
```diff
@@ -2114,7 +2114,7 @@
         ],
         "nodes": [
             {
-                "url": "wss://mandala.polkawallet.io",
+                "url": "wss://wss://rpc.pinknode.io",
```
Comment (Asmadek) [inline chains/v3/chains_dev.json:2117]: double `wss://`

**Review by Asmadek — CHANGES_REQUESTED** [review]

File: chains/v3/chains_dev.json (hunk 1 above)
Comment (Asmadek) [reply chains/v3/chains_dev.json:2117]: wss://rpc.pinknode.io/mandala

Comment (Asmadek) [reply chains/v3/chains_dev.json:2117]: or `wss://acala-mandala.api.onfinality.io/public-ws`

Comment (pgolovkin) [reply chains/v3/chains_dev.json]: Thank you, fixed

**Pull request closed without merging**